example/provider-local/shoot.yaml

```diff
@@ -12,7 +12,11 @@ spec:
   secretBindingName: local # dummy, doesn't contain any credentials
   region: local
   networking:
-    type: cilium
+    type: calico
+    # TODO(scheererj): Drop this once v1.32 has been released and https://github.com/gardener/gardener-extension-networking-calico/pull/250 is available as release
+    providerConfig:
+      apiVersion: calico.networking.extensions.gardener.cloud/v1alpha1
+      kind: NetworkConfig
```

Comment:
Earlier, we set https://github.com/gardener/gardener/blob/6d5cee235c53268564e109416b4ce068ec65e07f/example/provider-local/shoot.yaml#L19-L21

This is no longer needed now because of the configuration change for calico in the kind cluster?

Reply:
We now use ordinary `ip-in-ip` overlay and `bird` as backend. Therefore `backend: bird` would be the correct setting, which is the default and does not need to be specified.
Disabling `typha` as was done before is possible if desired. For now, I just moved to the default configuration as it works. If you prefer to run without `typha`, I can adapt accordingly.

Reply:
Sounds good, no need to adapt for now. Let's go with this!